File: create_config.c
// Random C struct generated on 2025-09-01
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

typedef struct {
    char name[100];
    int value;
    int active;
    long timestamp;
} CheckRequest;

CheckRequest* create_checkrequest(const char* name, int value) {
    CheckRequest* obj = malloc(sizeof(CheckRequest));
    if (obj == NULL) return NULL;
    
    strncpy(obj->name, name, sizeof(obj->name) - 1);
    obj->name[sizeof(obj->name) - 1] = '\0';
    obj->value = value;
    obj->active = 1;
    obj->timestamp = 1759861614;
    
    return obj;
}

void update_checkrequest_value(CheckRequest* obj, int new_value) {
    if (obj != NULL) {
        obj->value = new_value;
    }
}

void print_checkrequest(const CheckRequest* obj) {
    if (obj != NULL) {
        printf("Name: %s, Value: %d, Active: %d\n", 
               obj->name, obj->value, obj->active);
    }
}

void free_checkrequest(CheckRequest* obj) {
    if (obj != NULL) {
        free(obj);
    }
}

int main() {
    CheckRequest* obj = create_checkrequest("sample", 26);
    print_checkrequest(obj);
    free_checkrequest(obj);
    
    return 0;
}

File: create_config.go
// Random Go struct generated on 2025-09-30
package main

import (
    "encoding/json"
    "fmt"
    "time"
)

type ProcessValue struct {
    Name      string    `json:"name"`
    Value     int       `json:"value"`
    Active    bool      `json:"active"`
    CreatedAt time.Time `json:"created_at"`
}

func NewProcessValue(name string, value int) *ProcessValue {
    return &ProcessValue{
        Name:      name,
        Value:     value,
        Active:    true,
        CreatedAt: time.Now(),
    }
}

func (s *ProcessValue) UpdateValue(newValue int) {
    s.Value = newValue
}

func (s *ProcessValue) ToJSON() ([]byte, error) {
    return json.Marshal(s)
}

func main() {
    obj := NewProcessValue("sample", 33)
    data, _ := obj.ToJSON()
    fmt.Println(string(data))
}
